src/wayland/handlers/input_method.rs

```diff
@@ -32,6 +32,8 @@ impl InputMethodHandler for State {
             .map(|e| e.geometry())
             .unwrap_or_default()
     }
+
+    fn popup_repositioned(&mut self, _: PopupSurface) {}
```

Comment:
We probably should handle this similarly to how niri does it, no?

Reply:
Probably. I haven't looked at how niri implements it.

Reply:
https://github.com/YaLTeR/niri/pull/295